src/Microsoft.DotNet.Build.Tasks/PackageFiles/CodeCoverage.targets

```diff
@@ -50,6 +50,39 @@
     </PropertyGroup>
   </Target>
 
+  <Target Name="GenerateWindowsPdbsForAssemblyBeingTested" BeforeTargets="GenerateTestExecutionScripts">
+    <!-- As of 10/2017 OpenCover does not support portable PDBs, but we want the builds to generate
+         portable PDBs.  Thus we generate windows PDBs from portable PDBs here.   Can be removed
+         when OpenCover directly supports Portable PDBs (probably by early 2018) -->
+
+    <!-- We look for the DLL being tested for coverage and its PDB create a WindowsPDB\*.pdb which has
+         the windows PDB.  -->
+    <ItemGroup>
+      <PortableDllsToConvert Include="$(NETCoreAppTestSharedFrameworkPath)$(AssemblyBeingTestedName).dll">
+        <PdbPath>$(NETCoreAppTestSharedFrameworkPath)$(AssemblyBeingTestedName).pdb</PdbPath>
+        <TargetPath>$(NETCoreAppTestSharedFrameworkPath)WindowsPDB/$(AssemblyBeingTestedName).pdb</TargetPath>
+      </PortableDllsToConvert>
+
+      <ExistingPortableDllsToConvert Include="@(PortableDllsToConvert)" Condition="Exists('%(PortableDllsToConvert.Identity)') AND Exists('%(PortableDllsToConvert.PdbPath)')"/>
+    </ItemGroup>
+
+    <!-- There are two file in CoreFX, (System.Security.Permissions and Microsoft.Win32.Registry.AccessControl.pdb)  
+         are generated by the GenFacades task and lack a Debug diretory (see https://github.com/dotnet/buildtools/issues/1739)
+         This causes ConvertPortablePdbsToWindowsPdbs to fail.    Work around it right now by using WarnAndContinue 
+         Soon we won't need this code, (since OpenCover will support portable PDBs) so it is OK to just give up for now.  -->
+    <ConvertPortablePdbsToWindowsPdbs Files="@(ExistingPortableDllsToConvert)" ContinueOnError="WarnAndContinue"/>
```

Comment:
Is the known AV issue (https://github.com/dotnet/buildtools/issues/1737) likely to cause much pain in this context? I don't know when code coverage runs and who failures here would impact.

Here's a the retry workaround if you don't have a link handy: https://github.com/dotnet/buildtools/blob/e639cfa061b4f490f60baabc499123eb826055b6/src/Microsoft.DotNet.Build.Tasks/PackageFiles/Symbols.targets#L248-L273.

Reply:
I would liked to avoid preemptively working around #1737.    If an AV happens my expectation is that conversion of the PDB will fail (note the WarnOnError).  This means code coverage is likely to produce no results, but only for that one DLL, and it is not likely to be the same DLL each time.  

Finally this fix itself should only be useful for a few months.  I think it is OK to simply live with this in the mean time (or fix #1737)

Reply:
I'm pretty sure the AV crashes the entire process in an uncatchable way. At least, on Desktop people say it's a bad idea, and on Core the API doesn't seem to exist.

I don't know who it will impact, but living with the flakiness is ok to me.

Reply:
You are right that WarnOnError will probably not work.  It will probably crash the whole coverage run.   

I am willing to monitor it to see how it goes, if it happens once a week (or even once a month), I will react to it.